0001_initial_schema.down.sql

```diff
@@ -1,5 +1,9 @@
 BEGIN
 
-    CREATE TABLE 
+    DROP TABLE IF EXISTS Clothes
 
-END;
\ No newline at end of file
+    DROP TABLE IF EXISTS Outfits
+
+    COMMIT;
```

Comment:
Are we sure we need the COMMIT command? I'm referencing 
![image](https://user-images.githubusercontent.com/15041539/152558825-b7b2fe5b-0aa0-4f7f-aa5c-f37fe22735e9.png)

from this site: [https://www.tutorialspoint.com/sql/sql-transactions.htm](url)

Reply:
we can try it without it i can change